### PR TITLE
Tutorial and landing page

### PR DESCRIPTION
**Pull request by jo-pol**
See https://groups.io/g/GroundForge/message/233

### CODE REVIEW

File: docs/help/Binche.md [hunk 1]
```diff
@@ -67,4 +67,10 @@ Newbies may skip the bold steps when first trying to understand the scenario.
...
+ 
+In 2004, Hildegund Jene published a sampler called [_Binche Kompakt_](https://www.deutscher-kloeppelverband.de/katalog/) which featured 32 different grounds drawn from Flemish, Parisian,and Valenciennes laces.  Ground number 31 is the challenging TT4 ground described by Verbeke-Billiet.  When Jo Pol started to work on Binche Kompakt, she could find neither a thread diagram, nor a high resolution photo of a sample of TT4.  She decided to create a thread diagram using GroundForge.  This exercise motivated Pol to include several of the features you find in GroundForge today.
```
Comment (jo-pol) [inline docs/help/Binche.md:76]: You are repeating line 10.

File: docs/help/index.md [hunk 2]
```diff
@@ -1,43 +1,31 @@
 ---
 layout: default
-title: Help Intro
+title: Getting Started
```
Comment (jo-pol) [inline docs/help/index.md:3]: Did you notice that this title appears in the tab of the (Firefox) browser. As a general remark: therefore we might need a shorter one here than below.

File: docs/index.md [hunk 3]
```diff
@@ -5,38 +5,36 @@ title: GroundForge - Home
...
 
-Select a pair diagram from one of the [catalogue](#pattern-catalogues) pages or create your own.
-Choose stitches for thread diagram variations and toggle the thread colors
-to investigate the paths for contrasting threads. 
-A [tutorial](help) explains how. 
+What is GroundForge?
+--------------------
 
-Below two pair diagrams, each with four examples of thread diagrams.
+GroundForge is a tool that complements existing lace ground [catalogues](#pattern-catalogues) by making it possible to play with their designs.
```
Comment (jo-pol) [inline docs/index.md:11]: Perhaps also something short about printed catalogues/books.

File: docs/help/index.md [hunk 4]
```diff
@@ -1,43 +1,31 @@
...
-The tool also allows you to specify the combinations of stitches that are made when two pairs meet.  You can make changes and quickly observe how this affects the thread diagram.
-This greatly speeds up the process of experimenting with lace grounds; no need to wind bobbins or prepare prickings. The final test is always to make a hand-worked sample&mdash;only in a physical sample can the interactions between pins and threads under tension be completely observed&mdash;however, GroundForge can help you narrow the number of samples you need to test by hand.
-You can experiment and play around with the hundreds of patterns provided here, or you can create your own.
-Even when reinventing the wheel, your own discoveries can bring great joy.
-
-
 Virtual tour
 ------------
 
 Take a walk through the various catalogues on the GroundForge website in this [screencast](https://github.com/d-bl/GroundForge/releases/download/2019-Q2/catalogues.mp4) (MP4 format, 123 MB, 2:32 min). You will notice that Rose ground, and many variations related to it, appears in each of the catalogues.
```
Comment (jo-pol) [inline docs/help/index.md:25]: The two spaces at the end of this line cause a line break in the decorated version of the page. Is that intentional?

File: docs/help/index.md [hunk 5]
```diff
@@ -57,7 +45,7 @@ Open the [diagram] for ground G-2 (column G, row 2) in the [Whiting catalogue].
 
 ### Highlight a thread
 
-In the thread diagram on the far right, click on the gray square at the start of a thread.  You may need to scroll to the top of the diagram to see it.  The colour of the thread will change from black to red.  Click on the square again and the thread colour returns to black.  You can highlight more than one thread at a time.  For more colours and options, visit [thread properties](Thread-Properties).
+In the thread diagram on the far right, click on the gray square at the start of a thread or the black bobbin at the end of a thread.  You may need to scroll the diagram to see the ends of a thread.  The colour of the thread will change from black to red.  Click on the square or bobbin again and the thread colour returns to black.  You can highlight more than one thread at a time.  For more colours and options, visit [thread properties](Thread-Properties).
 
 ![](images/G2-toggle-thread.png)
```
Comment (jo-pol) [inline docs/help/index.md:50]: The image is too large fit on a reasonable monitor together with the text. It requires too much scrolling to understand.

**Review by jo-pol — COMMENTED** [review]
A few comments. Did not read everything very thoroughly. In general great improvements.

At the same time I'm curious of consequences now that I created the pull request. Never done it that way before.

File: docs/index.md [hunk 6]
```diff
@@ -5,38 +5,36 @@ title: GroundForge - Home
...
-to investigate the paths for contrasting threads. 
-A [tutorial](help) explains how. 
+What is GroundForge?
+--------------------
 
-Below two pair diagrams, each with four examples of thread diagrams.
+GroundForge is a tool that complements existing lace ground [catalogues](#pattern-catalogues) by making it possible to play with their designs. 
+* It generates an interactive thread diagram in which you can highlight the path of a thread; a useful feature when working with multiple thread colours or thicknesses in a design.  
+* You can specify the combinations of stitches to make when two pairs meet.  This is very useful for communicating and documenting a new lace ground.
+* Experiment with lace grounds without the need to wind bobbins or prepare a pricking. The final test is always to make a hand-worked sample&mdash;only in a physical sample can the interactions between pins and threads under tension be completely observed&mdash;however, GroundForge can help you narrow the number of samples you need to test by hand.
```
[the remark this answers is not in the document]
Comment (veronika) [reply docs/index.md:14]: The `&mdash;` marks are around the sub-clause "only in a physical sample can the interactions between pins and threads under tension be completely observed" so they were correct in the original.  They are not a hyphen between hand and worked.

File: docs/help/index.md [hunk 7]
```diff
@@ -57,7 +45,7 @@ Open the [diagram] for ground G-2 (column G, row 2) in the [Whiting catalogue].
 
 ### Highlight a thread
 
-In the thread diagram on the far right, click on the gray square at the start of a thread.  You may need to scroll to the top of the diagram to see it.  The colour of the thread will change from black to red.  Click on the square again and the thread colour returns to black.  You can highlight more than one thread at a time.  For more colours and options, visit [thread properties](Thread-Properties).
+In the thread diagram on the far right, click on the gray square at the start of a thread or the black bobbin at the end of a thread.  You may need to scroll the diagram to see the ends of a thread.  The colour of the thread will change from black to red.  Click on the square or bobbin again and the thread colour returns to black.  You can highlight more than one thread at a time.  For more colours and options, visit [thread properties](Thread-Properties).
```
[the remark this answers is not in the document]
Comment (veronika) [reply docs/help/index.md:48]: Yes, I like these changes.  The only thing that I might change is "Another click at the same square or bobbin again and the thread colour returns to black."  It doesn't really matter whether you click on the square or the bobbin.  So I might reword this as:

"Click on the square or bobbin of a red thread and it returns to black."

File: docs/help/Advanced.md [hunk 8]
```diff
@@ -102,61 +102,64 @@ For convenience, you can set all stitches in the pattern to the same value.  In
...
+
+[G-4 with left footside]: /GroundForge/tiles?patchWidth=28&patchHeight=40&l1=ctctt&i1=ctc&h1=ctc&g1=ctc&f1=ctc&d1=ctctt&j2=ctcrr&l3=ctctt&i3=ctc&h3=ctc&g3=ctc&f3=ctcll&d3=ctctt&q4=ctctt&o4=ctctt&m4=ctctt&k4=ctctt&i4=ctt&g4=ctcll&f4=ctcll&e4=ctctt&c4=ctctt&a4=ctctt&j5=ctctt&f5=ctctt&q6=ctctt&m6=ctctt&c6=ctctt&p7=c&n7=ctc&m7=ctc&l7=ctctt&j7=ctctt&h7=ctctt&f7=ctctt&d7=ctctt&c7=ctctt&c9=ctctt&a9=ctctt&c11=ctctt&c12=ctctt&c14=ctctt&a14=ctctt&footside=X--,X--,---,G-E,XW-,X-C,X-1,-Y-,B-B,XW-,X-C,X-4,-Y-,B-N&tile=5-m998-z5-----,------5-------,g-aaab-wd-----,-246-mv6-o-k-e,--5---5---y-w-,---w-y---b---c,h-g-5-n-l3h-e-,&footsideStitch=ctctt&tileStitch=ctc&headsideStitch=ctctt&shiftColsSW=-7&shiftRowsSW=7&shiftColsSE=7&shiftRowsSE=7
 
 Thread diagram as pair diagram
 ------------------------------
+**This feature is still under construction.  We welcome any suggestions on how to improve it.**
```
[the remark this answers is not in the document]
Comment (veronika) [reply docs/help/Advanced.md:144]: Users can also provide valuable feedback on UI design, not just "experts".  Unless we are overwhelmed with feedback, I think it is OK to keep this statement general.

File: docs/help/index.md (hunk 4 above)
Comment (veronika) [reply docs/help/index.md:25]: Not intentional.

File: docs/help/index.md (hunk 5 above)
Comment (veronika) [reply docs/help/index.md:50]: I will resize.

File: docs/help/Binche.md (hunk 1 above)
Comment (veronika) [reply docs/help/Binche.md:76]: Line 10 is a much shorter version of line 76.  Since this history comes at the bottom of the web page and explains the background, I think it is still of interest and worth keeping.

File: docs/help/index.md (hunk 5 above)
Comment (jo-pol) [reply docs/help/index.md:50]: I didn't mean plain resizing. These two images show four purposes. The handles, the pop-ups, thread behavior inside the pattern/stitch and along the edge. The bug at the bottom with the bobbins two by two in the wrong order might be confusing.

Screenshot 2020-09-13 at 08 36 51

Comment (jo-pol) [reply docs/help/index.md:50]: Now that I study the top of these diagrams: show one or two rows and columns more and two diagram variants. One variant with two twists between the stitches and one with a single twist. I'll leave it up to you to stick to two diagrams or make it four. plenty of room to have four side by side. It shows how the threads change roles at the edge or not depending on an odd or even number of twists. Roles as in inside/outside-threads. Note that you can play with the zoom factor of your browser to play with the ration of the pop-up size and handle-size. At least in firefox the pop-up doesn't increase along.

Comment (veronika) [reply docs/help/index.md:50]: I have read over the tutorial again and I have to disagree with changing to a simpler ground.  The entire tutorial is about playing with G2 from Whiting's sampler. A person following along with the tutorial will expect to see G2, not Brussel's ground.

Comment (veronika) [reply docs/help/index.md:50]: One option is to re-write this whole "Getting started" tutorial with a simpler example.  This might not be a bad idea.  I was trying to keep the tutorial close to the original Binche Kompakt 31 example.  However, now that Binche Kompakt 31 has a home of its own, a simple example may be more useful as a starting point.

File: docs/help/Advanced.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (jo-pol) [reply docs/help/Advanced.md:144]: Problem is I'm most likely not able to implement without an expert.

File: docs/help/index.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (jo-pol) [reply docs/help/index.md:48]: Even better. I didn't dare to change too much.

**Comment by veronika** [comment]
I have pushed out new updates following Jo's feedback.  The changes are minor.  I will rewrite the Getting Started tutorial with a simple example after the current changes have been pulled into master.
